src/ResourceIndexer.js

```diff
@@ -18,7 +18,7 @@ export default class {
    * Builds up an index entry out of a JSON body, given index field mappings from config
    * @returns {Object} an object containing configured field values if any found
    */
-  index() {
+  async index() {
```

Comment:
I'm assuming this should have been `async` before because there's nothing that makes me think it is changing to async now.